backend/src/idl/policy.proto

```diff
@@ -0,0 +1,54 @@
+syntax = "proto3";
+
+package policy;
+
+option java_multiple_files = true;
+
+message PolicyDetails {
+    WorkSchedule policy = 1;
```

Comment:
As per  [release notes of v3.0.0](https://github.com/google/protobuf/releases/tag/v3.0.0),  Explicit "optional" keyword are disallowed in proto3 syntax, as fields are optional by default; required fields are no longer supported.